src/searchPage/filterPanel/filterPanel.html

```diff
@@ -0,0 +1,3 @@
+<nav class="sidebar">
```

Comment:
Why use nav here? Also, this might not always be a sidebar, if we want to reuse it

Reply:
changed to ul list